File: Modul-3/Soal-4/PRAK304-2410817210016-MUHAMMADRIZKYAKBAR.py
Value = int(input("Masukkan Nilai: "))

if Value >= 100:
    print("Anda Menginput Melebihi Limit Bilangan")
elif Value == 0:
    print("Nol")
elif Value >= 20:
    print("Puluhan")
elif Value >= 10:
    print("Belasan")
elif Value >= 1:
    print("Satuan")
else:
    print("Invalid")

File: Modul-3/Soal-4/PRAK304-2410817210016-MUHAMMADRIZKYAKBAR.c
#include <stdio.h>

int main(){

    int Value;

    printf("Masukkan Nilai: ");
    scanf("%d", &Value);

    if (Value >= 100){printf("Anda Menginput Melebihi Limit Bilangan");}
    else if (Value >= 20){printf("Puluhan");}
    else if (Value >= 10){printf("Belasan");}
    else if (Value >= 1){printf("Satuan");}
    else if (Value = 0){printf("Nol");}
    else {printf("invalid");}
    return 0;
}